### PR TITLE
[docs] Add language-swift

### PR DESCRIPTION
**Pull request by modocache**

### CODE REVIEW

**Comment by facebook-github-bot-7** [comment]
By analyzing the blame information on this pull request, we identified @JoelMarcey, @tyangliu and @zertosh to be potential reviewers.

**Comment by zertosh** [comment]
@JoelMarcey is the person to review this!

**Comment by modocache** [comment]
Updated the pull request after running the images through [ImageOptim](https://imageoptim.com/).

**Comment by facebook-github-bot-9** [comment]
@modocache updated the pull request.

**Comment by zertosh** [comment]
lgtm, but just one tiny thing: Can you add the same disclaimer as https://github.com/facebook/nuclide/blame/655190a/pkg/nuclide-swift/README.md#L6 ?

**Comment by modocache** [comment]
@zertosh Good point. Done!

**Comment by facebook-github-bot-9** [comment]
@modocache updated the pull request.

**Comment by zertosh** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-6** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/308752622792945/int_phab) to review internal test results.

**Comment by JoelMarcey** [comment]
@zertosh Just saw this. Sorry.

@modocache There are few minor grammar and formatting issues that I would have recommended updating, but nothing major for sure. I am glad to see this!

**Comment by modocache** [comment]
@JoelMarcey Anything in particular? I'll run spell-check on it and submit fixes for what I find. :)

**Comment by JoelMarcey** [comment]
Oh the spelling was just fine I think :)

Just some minor, nitty re-wording. I can comment in this PR.

File: docs/_docs/languages/swift.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+<br/>
+
+* TOC
+{:toc}
+
+## Configuring Nuclide for Swift Package Development
+
+Nuclide will attempt to find a Swift executable automatically:
+
+- On macOS, it will use the latest version of Swift installed, at `/Library/Developer/Toolchains/swift-latest.xctoolchain`.
```
Comment (JoelMarcey) [inline docs/_docs/languages/swift.md]: no "," here.

File: docs/_docs/languages/swift.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+- On Linux, it will find a Swift executable based on your PATH.
```
Comment (JoelMarcey) [inline docs/_docs/languages/swift.md]: Wondering if PATH should be `PATH`?

**Comment by JoelMarcey** [comment]
Reading it again, I think that is it. Like I said, minor :-)

File: docs/_docs/languages/swift.md (hunk 1 above)
Comment (modocache) [inline docs/_docs/languages/swift.md]: You mean: "On macOS it will use"? Can do.

Comment (JoelMarcey) [inline docs/_docs/languages/swift.md]: No sorry, I meant the second comma.

**Pull request closed without merging**